esmvaltool/preprocessor/_mask.py

```diff
@@ -86,7 +153,7 @@ def _apply_fx_mask(fx_mask, var_data):
 
 def mask_landsea(cube, fx_files, mask_out):
     """
-    Mask out either land or sea
+    Mask out either land or sea.
```

Comment:
This sentence is the same as the one below, one of them can be deleted.

Reply:
nono, it's there as a title, only difference now it has a full stop since pep8 rules :grin: